README.md

```diff
@@ -1 +1,7 @@
-# aql-editor-backend-spring-boot-starter
\ No newline at end of file
+# Spring boot starter for the AQL editor backend
+
+Added as dependency the starter will expose the following REST endpoints:
+
+- /aqleditor/v1/aql
+- /aqleditor/v1/containment
+- /aqleditor/v1/template
```

Comment:
you changed the path. which means the backend postman Collection no longer works
see https://github.com/ehrbase/aql-editor-backend/pull/10#pullrequestreview-571949299

Reply:
changed in aql-editor-backend